src/main/java/org/squiddev/plethora/integration/astralsorcery/IntegrationAstralSorcery.java

```diff
@@ -0,0 +1,172 @@
+
+			return Collections.singletonMap("crystalProperties", out);
+		}
+
+		@Nullable //If it returns Null, Astral broke something...
+		@Override
+		public CrystalProperties getExample() {
+			return CrystalProperties.getMaxRockProperties();
+		}
+	};
+
+	public static final IMetaProvider<IConstellation> META_I_CONSTELLATION = new BasicMetaProvider<IConstellation>(
+		"Provides the name, colors, and tier of an IConstellation"
+	) {
+
+		@Nonnull
+		@Override
+		public Map<String, ?> getMeta(@Nonnull IConstellation context ) {
+			Map<String, Object> out = new HashMap<>(8);
+
+			String translationKey = context.getUnlocalizedName();
+			out.put("name", translationKey);
+			out.put("simpleName", context.getSimpleName());
+			out.put("displayName", Helpers.translateToLocal(translationKey));
+
+			out.put("color", context.getConstellationColor().getRGB()); //Used for particles on rituals, collectors?
+			out.put("colour", context.getConstellationColor().getRGB());
+			out.put("tierColor", context.getTierRenderColor().getRGB());
+			out.put("tierColour", context.getTierRenderColor().getRGB());
+
+			out.put("tier", getConstellationTier(context));
+
+			//Exposing the stars and the connections wouldn't help players much, unless they want to visualize
+			// constellations on a computer...?
+
+			//Not wrapping in a namespace, as IConstellation is _usually_ a field on other objects
+			return out;
+		}
+
+		@Nullable
+		@Override
+		public IConstellation getExample() {
+			return Constellations.discidia;
+		}
+	};
+
+	//REFINE As this is only ever called via META_RESPLENDENT_PRISM, we technically could just use a method
+	// rather than a full-fledged IMetaProvider
+	public static final IMetaProvider<AmuletEnchantment> META_AMULET_ENCHANTMENT = new BaseMetaProvider<AmuletEnchantment>(
+		"Provides the type, level, and boosted enchantment (if applicable) of this Amulet Enchantment.\n" +
+			"For examples, see META_RESPLENDENT_PRISM"
+	) {
+
+		@Nonnull
+		@Override
+		public Map<String, ?> getMeta(@Nonnull IPartialContext<AmuletEnchantment> context ) {
+			Map<String, Object> out = new HashMap<>(4);
+			AmuletEnchantment target = context.getTarget();
+
+			DynamicEnchantment.Type enchantType = target.getType();
+			out.put("bonusType", enchantType.toString());
+
+			//REFINE I feel like I'm missing some simple logical fallacy here...
+			// Could just be that the code is brittle, as broken data could cause a type other than
+			// `ADD_TO_EXISTING_ALL` to have a `null` enchant...
+			Enchantment enchant = target.getEnchantment();
+			if (enchantType.hasEnchantmentTag() && enchant != null) {
+				out.put("boostedEnchant", enchant.getName());
+				out.put("boostedEnchantFullName", enchant.getTranslatedName(1));
+			} else {
+				out.put("boostedEnchant", "all");
+			}
+
+			out.put("bonusLevel", target.getLevelAddition());
+
+			return out;
+		}
+
+		@Nullable
+		@Override
+		public AmuletEnchantment getExample() {
```

Comment:
Probably worth just removing this method as it's the same as the parent class.

Reply:
Hmmm... is it bad that my first thought is to check for an inspection to highlight it for me?  I mean, at least for code this simple, checking if it does the same thing as the method it is overriding should be trivial...